CONTRIBUTING.rst

```diff
@@ -108,6 +113,6 @@ Before you submit a pull request, check that it meets these guidelines:
 1. The pull request should include tests and must not decrease test coverage.
 2. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a docstring.
-3. The pull request should work for Python 2.7, 3.3, 3.4 and 3.5. Check
-   https://travis-ci.com/inveniosoftware/invenio-collections/pull_requests
+3. The pull request should work for Python 3.6, 3.7, 3.8 and 3.9. Check
```

Comment:
Can we switch this to "The pull request should work for Python versions supported by Invenio."? We might want to do a global change on this....since many of them still mention Python 2.7

Reply:
We've got it [here](https://inveniordm.docs.cern.ch/install/requirements/), and CONTRIBUTING links below to the CI list, which is probably more likely to be updated anyway.